### DISCUSSION TITLE
Is there any public communication channel?

### DISCUSSION THREAD

**Discussion by seonWKim** [tursodatabase/turso#300] [Q&A] [answered]
I wonder if there is any communication channel used to develop or Q&A for the limbo project. 
It would be great for the contributors to be able to communicate with the maintainers when solving the issues etc

**Comment by penberg** [comment, marked as answer]
Hey @seonWKim! I added a Discord link to README.md where people can join to discuss with developers.

**Reply by seonWKim** [reply]
Thank you so much!
